inference-engine/tests/ie_test_utils/functional_test_utils/layer_test_utils.cpp

```diff
@@ -99,6 +131,38 @@ void LayerTestsCommon::Infer() {
     inferRequest.Infer();
 }
 
+std::vector<InferenceEngine::Blob::Ptr> computeCpuTarget(const InferenceEngine::CNNNetwork cnnNetwork, const std::vector<InferenceEngine::Blob::Ptr> inputs) {
```

Comment:
All of the written code already exists in `ConfigurePlugin()`,` LoadNetwork()` and `Infer()` methods. Please reuse it instead of adding a new implementation. 
Also, `core` is a class attribute assigned in the constructor, why do you create local `CPUcore`  object?

Reply:
The model I'd like to test contains TensorIterator which is not yet implemented in ngraph interpreter. The fastest way, AFAIK, to compare GNA output to check its correctness was to compare it with CPU. The core assigned with a constructor uses targetDevice, in this case GNA, not CPU, that's why I create a local one, but I can reimplement the comparison to use different method.